frameworks/Nim/nim-stdlib/src/handlers.nim

```diff
@@ -0,0 +1,106 @@
+import asynchttpserver, asyncdispatch, times, random, db_postgres, cgi, strtabs, strutils, json, algorithm
+
+import defs
+
+proc handleHTTPErrors*(req: Request, code: HttpCode, msg: string) {.async.} =
+    let headers = {"Date": now().utc.format("ddd, dd MMM yyyy HH:mm:ss") & " GMT", "Content-type": "text/plain; charset=utf-8"}
+    await req.respond(code, msg, headers.newHttpHeaders())
+
+proc handlePlaintext*(req: Request) {.async.} =
+    let headers = {"Date": now().utc.format("ddd, dd MMM yyyy HH:mm:ss") & " GMT", "Content-type": "text/plain; charset=utf-8",  "Server": "Example"}
+    await req.respond(Http200, "Hello, World!", headers.newHttpHeaders())
+
+proc handleJson*(req: Request) {.async.} =
+    let headers = {"Date": now().utc.format("ddd, dd MMM yyyy HH:mm:ss") & " GMT", "Content-type": "application/json; charset=utf-8", "Server": "Example"}
+    await req.respond(Http200, """{"message":"Hello, World!"}""", headers.newHttpHeaders())
```

Comment:
This isn't acceptable for the JSON test requirements. An object has to be created and then serialized

Reply:
@nbrady-techempower thanks. I have fixed all instances of this issue. now all json objects are created using Nim's serializer.

Reply:
To help with the review, Nim uses "%*" as the operator for creating json. More here
https://nim-lang.org/docs/json.html#creating-json